src/calculateRentalCost.js

```diff
@@ -4,7 +4,22 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const RENT_PER_DAY = 40;
+  const TOTAL = days * RENT_PER_DAY;
```

Comment:
The variable 'TOTAL' should be declared with 'let' instead of 'const' since its value might change later based on the applied discounts.